### DISCUSSION TITLE
Post-Hoc compare multiple Kaplan-Meier multiple curves

### DISCUSSION THREAD

**Discussion by weihevt** [CamDavidsonPilon/lifelines#1460] [Q&A] [answered]
Hi,
Is there a function in lifelines comparing multiple Kaplan-Meier curves and can return the post-hoc p-values for each two curves?

Thanks

**Comment by CamDavidsonPilon** [comment, marked as answer]
The closest thing would be [pairwise_logrank_test](https://lifelines.readthedocs.io/en/latest/lifelines.statistics.html#lifelines.statistics.pairwise_logrank_test). 

See docs about it [here](https://lifelines.readthedocs.io/en/latest/Examples.html?highlight=pairwise_logrank_test#logrank-test).

**Comment by weihevt** [comment]
Got it. Thanks so much.
